codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/file/Options.kt

```diff
@@ -30,28 +30,29 @@ import com.google.protobuf.BoolValue
 import com.google.protobuf.StringValue
 import io.spine.protobuf.pack
 import io.spine.protodata.Option
+import io.spine.protodata.option
 
 /**
  * The option to instruct `protoc` to generate multiple Java files.
  */
-public val javaMultipleFiles: Option = Option.newBuilder().apply {
+public val javaMultipleFiles: Option = option {
     name = "java_multiple_files"
     value = BoolValue.of(true).pack()
-}.build()
+}
 
 /**
  * Obtains an option to set the Java package with the given [name]
  * for the generated code.
  */
-public fun javaPackage(name: String): Option = Option.newBuilder().apply {
+public fun javaPackage(name: String): Option = option {
     this.name = "java_package"
     value = StringValue.of(name).pack()
-}.build()
+}
 
 /**
  * Obtains the option to set the [name] of the outer Java class.
  */
-public fun javaOuterClassName(name: String): Option = Option.newBuilder().apply {
+public fun javaOuterClassName(name: String): Option = option {
```

Comment:
@alexander-yevsyukov, does your IDE work fine in here? Mine shows an error sometimes.

Reply:
@dmdashenkov, this is because deduplication of generated sources does not work in ProtoData. If you delete `build/generated/source/proto/kotlin` the error goes away.

Strangely, the deduplication works OK in `mc-java`. We tried to find out the difference between two projects with @armiol, but found nothing so far.

The only reason I can suggest now is that `GenerateProtoTask.excludeProtocOutput()` from `io.spine.protodata.gradle.plugin.Plugin.kt` is executed _earlier_ than all the tasks are configured in ProtoData Gradle project. And in `mc-java` timing is more favorable to us.